store/sqlstore/supplier.go

```diff
@@ -1178,6 +1183,10 @@ func (ss *SqlSupplier) CheckIntegrity() <-chan store.IntegrityCheckResult {
 	return results
 }
 
+func (ss *SqlSupplier) UpdateLicense(license *model.License) {
+	ss.license = license
```

Comment:
Given that we read and write to this value from multiple threads, should we coordinate with a mutex and avoid the eventual race detector bug?

Reply:
Yes, I can easily add a mutex there